CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ v0.2.0 (unreleased)
     ThriftRW now provides a plugin system to allow customizing code generation.
     Initially, only the generated code for `service` declarations is
     customizable. Check the documentation for more details.
+-   Breaking: Fixed a bug where all-caps attributes that are not known
+    abbreviations were changed to PascalCase.
+-   Breaking: The `String()` method for `enum` types now returns the name of
```

Comment:
Done https://github.com/thriftrw/thriftrw-go/pull/217

(I think the all-caps is unnecessary but the bold works.)